src/index.html

```diff
@@ -1,13 +1,59 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img 
+        src="images/imac.jpeg"
+        alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+        class="card__image"
+      >
+      
+      <h2 class="card__name">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <div class="card__code">
+        Product code: 195434
+      </div>
+
+      <div class="card__review review">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <p class="review__reviews">Reviews: 5</p>
+      </div>
+
+      <div class="card__price price">
+        <p class="price__text">Price:</p>
+        <p class="price__sum">$2,199</p>
+      </div>
+
+      <button class="button">
+        <a
+          href="/"
+          class="button__link"
+          data-qa="hover">
+          buy
+        </a>
```

Comment:
```suggestion
        <a
          href="/"
          class="button__link"
          data-qa="hover"
        >
          buy
        </a>
```
Closing tag should be under the opening